src/codeu/chat/common/LogoutCallback.java

```diff
@@ -0,0 +1,4 @@
+package codeu.chat.common;
+public interface LogoutCallback{
```

Comment:
Wee need to implement the callback in order to pass the function to the other panels, it is a Java standard to create callbacks using interfaces.